ddtrace/internal/utils/http.py

```diff
@@ -29,6 +29,7 @@
 from ddtrace.internal.constants import W3C_TRACESTATE_SAMPLING_PRIORITY_KEY
 from ddtrace.internal.utils import _get_metas_to_propagate
 from ddtrace.internal.utils.cache import cached
+import ddtrace.vendor.xmltodict as xmltodict
```

Comment:
there is a chance this is causing the rss slo breaches, moving the import to global means it is getting imported more eagerly and in scenarios it wouldn't get imported in before.

if you wanted to move this import back to L394, could be an interesting test.